doc/classes/VisualServer.xml

```diff
@@ -2242,17 +2253,6 @@
 				Sets the color of the shadow cast by the light. Equivalent to [member Light.shadow_color].
 			</description>
 		</method>
-		<method name="light_set_use_gi">
```

Comment:
Removing methods breaks the API compatibility, so cannot be merged in `3.2` as is.

Reply:
I updated the PR. I kept `light_set_use_gi` and marked it as deprecated, it should work the same as it did but it now it just calls `light_set_bake_mode` internally.